docs/usage.md

```diff
@@ -73,6 +73,8 @@ Helm args must be defined in the application manifest:
           value: -f values-dev.yaml -f values-dev-tag.yaml
 ```
 
+**Note: Bypassing the parameters like this can be dangerous in a multi-tenant environment as it could allow for malicous injection of arbitrary commands. So be cautious when doing something like in a production environment. Making sure that proper permissions and protections are very important when doing something liek this.** 
```

Comment:
```suggestion
**Note: Bypassing the parameters like this can be dangerous in a multi-tenant environment as it could allow for malicious injection of arbitrary commands. So be cautious when doing something like in a production environment. Making sure that proper permissions and protections are enforced is very important when doing something like this.** 
```